site/content/docs/playwright-checks/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Playwright Checks reference - Checkly Docs
+displayTitle: Getting started with Playwright Monitoring
+navTitle: Overview
+weight: 14
+slug: /
+menu:
+  resources:
+    parent: "Playwright checks (Alpha)"
+
+---
+
+With Checkly, you can convert your Playwright tests directly into scheduled monitors.
+
+You can schedule checks from different locations and trigger alerts for your team to act on when a critical flow or API fails in your product.
+
+## What's a Playwright Check?
+
+A Playwright check offers all of Playwright's features, natively:
+
+* Dependencies between tests or projects → `projects`, `dependencies`, `globalSetup`, `globalTeardown`, reused `StorageState`, `test.beforeEach`...
+* Test level retries.
+* Automatic flagging of flaky tests when passed on retry.
+* Fake media access: whether it's a QR in a video you are parsing or access to a microphone, you got it.
```

Comment:
Is there a validated appetite around this? I think this is very cool, but for a monitoring scenario, it's very niche IMO. 🫣

Reply:
haha, thank you, Stefan, so during the private alpha, the people that were really eager to try this out were the ones blocked by fake media access, so I'd say the appetite is really there and enticing?